dbt/sagerx/models/staging/fda_unfinished/_fda_unfinished__models.yml

```diff
@@ -6,7 +6,7 @@ models:
     columns:
       - name: ndc11
         description: "The ndcpackagecode field, normalized to a NDC11 format."
-        tests:
+        data_tests:
```

Comment:
What's the difference between `tests` and `data_tests`?  Should we add this to our style guide?